### DISCUSSION TITLE
민감정보가 담긴 application.yml 파일과 공개해도 되는 application.yml 파일을 분리하고 싶습니다!

### DISCUSSION THREAD

**Discussion by 02ggang9** [BDD-CLUB/01-doo-re-back#24] [논의 사항]
민감정보가 담긴 application.yml 파일과 공개해도 되는 application.yml 파일을 분리하고 싶습니다! 이유는 문서화 하면서 작성했습니다!

[문서](https://02ggang9.github.io/trouble/DuplicateKeyException/)

**Comment by shkisme** [comment]
[작성해주신 글](https://02ggang9.github.io/trouble/DuplicateKeyException/) 보고 왔습니다!

파일을 분리하여 CI/CD 코드를 가독성있게 작성하는 것과 효율적인 관리가 가능하다면 분리하는 것에 동의합니다!

**Comment by lcqff** [comment]
해결하셨군요! 축하드립니다.
저도 전과 같은 문제가 발생할 확률이 높고 분리하는 것이 보안적으로 우수하다면 분리하는 게 좋다고 생각합니다!

**Comment by amaran-th** [comment]
저도 민감정보가 담긴 배포 코드(dev, prod)의 yml 파일과 로컬 환경의 배포 코드를 분리하는 것에 찬성합니다~ 스프링 프로필을 활용해도 될 것 같네요.
그리고 저는 gitignore도 좋지만 서브모듈을 활용하는 편이 팀원 간에 yml 파일을 동기화하기 용이할 것 같다는 생각입니다!

**Comment by JJimini** [comment]
저도 분리하는게 더 좋을 것 같아요~! 수고하셨습니다!
